### DISCUSSION TITLE
[numjs] ndarray.slice should support null args

### DISCUSSION THREAD

**Discussion by Interpause** [DefinitelyTyped/DefinitelyTyped#59537] [Issues with a @types package] [answered]
Currently typing only allows args of `number` or `number[]`. However following https://github.com/nicolaspanel/numjs#slicing-and-striding, `null` should also be accepted.

[a comment by an automated account is not shown]

**Comment by MysteryBlokHed** [comment, marked as answer]
I just opened #59578

**Reply by Interpause** [reply]
thanks
